dev-proxy-plugins/Inspection/DevToolsPlugin.cs

```diff
@@ -23,6 +23,12 @@ public enum PreferredBrowser
 public class DevToolsPluginConfiguration
 {
     public PreferredBrowser PreferredBrowser { get; set; } = PreferredBrowser.Edge;
+
+    /// <summary>
+    /// Path to the browser executable.
+    /// If not set, the plugin will try to find the browser executable based on the PreferredBrowser.
+    /// </summary>
+    public string PreferredBrowserPath { get; set; } = string.Empty;
```

Comment:
Why do we need to ask folks for the path to the preferred browser? It seems excessive to ask for both the preferred browser and the path, as either one should be sufficient. Is there a specific scenario for which we need this?

Reply:
so the feature doesn't work on Linux since there's no browser path there - or I cannot find the browser.
If I install chrome and provide the path to it, it works.

So we should either throw an error if someone tries to run it in Ubuntu (like I'm using it) or find a way to find chrome there

Reply:
Understood. Is there a standard location where the browser install to on Linux? If so, we could update the logic in here to keep it consistent with other OSes: https://github.com/microsoft/dev-proxy/blob/9e73c68c7d85780d0820d745d0dd020cfff0f0c3/dev-proxy-plugins/Inspection/DevToolsPlugin.cs#L62-L76

Reply:
I don't think it is. I've tested with binary installed using playwright. 
Someone would have to add chrome to PATH, then we could reference it using just executable name.

This isn't the case for me on ubuntu on WSL, which kind of make sense, since it's just terminal

Reply:
I don't think that Playwright and WSL are a good comparison here. They're both headless. When you're working in WSL, you wouldn't run the browser in it, but rather start it on the host, right? Shouldn't we be checking for regular browser processes rather than processes temporarily spawned by Playwright or other similar tools?

Reply:
so yes and no.
I do most work on WSL - so that's when I need to setup proxy. It's a pain to go WSL -> windows host but not other way around. 
Even though WSL is headless - you can still open chrome, run playwright etc.

That `PreferredBrowserPath` is not a hill I want to die on :) 
I don't like the idea of providing path to browser as much as anyone else, but it was the only thing that worked for me on WSL. 
On the Ubuntu VM chrome might be in a system path, so I can just make that assumption and for everything non-OSX/Windows check for chrome binary in path and fail if it's not there? Then I could remove `PreferredBrowserPath`  and on WSL update my bash setting to include it?

here how it looks right now on WSL with preferred path:
![image](https://github.com/microsoft/dev-proxy/assets/3539908/c9367886-564c-4795-833f-1040f34f1ff3)

Reply:
I'm not against `PreferredBrowserPath` on principle, just trying to find a way to get it all to work with as little friction as possible 🙂  If we could provide a default path with the ability to override then it would be a nice middle ground between a default setup that works for most and an escape hatch in case you need something else.

Is there a reason you install the browser in WSL using Playwright? I'd expect the browser to be installed as an OS-wide app using a package manager like apt, but perhaps there are some good reasons to do it like you do.

I appreciate your feedback!

Reply:
I've updated the code with default browser path and added comment for install instructions